features/package-install.feature

```diff
@@ -1,5 +1,35 @@
 Feature: Install WP-CLI packages
 
+  Scenario: Install a package with an old package index url in package composer.json
+    Given an empty directory
+    And a composer.json file:
```

Comment:
If the package directory is in `package-directory` off the test directory, then you can just place `composer.json` there with:

```
And a package-directory/composer.json file:
```